src/main/java/seedu/address/model/listmanagers/packinglistitem/PackingListItem.java

```diff
@@ -13,31 +13,59 @@
 public class PackingListItem implements UniqueListElement {
     // Identity fields
     private final ItemName itemName;
+    private final ItemCategory itemCategory;
```

Comment:
Since `itemCategory` does not determine the uniqueness of an item (as seen from your `isSame()` method), it should be placed under the data fields as per your comments. Placing them under the right comment headings will help readers although it does not break your code.